packages/apps-config/src/api/params/inflation.ts

```diff
@@ -3,7 +3,7 @@
 
 import type { ApiPromise } from '@polkadot/api';
 
-import { DOCK_POS_TESTNET_GENESIS, KUSAMA_GENESIS, NEATCOIN_GENESIS, NFTMART_GENESIS, POLKADOT_GENESIS } from '../constants';
+import { DOCK_POS_TESTNET_GENESIS, KUSAMA_GENESIS, NEATCOIN_GENESIS, NFTMART_GENESIS, POLKADOT_GENESIS } from '../constants.js';
```

Comment:
Usually, it is done by webpack's `resolve.extensions` ([docs](https://webpack.js.org/configuration/resolve/#resolveextensions)). What option is preferred here?

Reply:
Thank you for your message! It appears that the issue you are experiencing is not related to Webpack, but rather with the TypeScript compiler. You received the error while running `yarn build`, which includes the command `tsc --build tsconfig.build.json`.

There was an error during the build: 
```sh
error TS2835: Relative import paths need explicit file extensions in EcmaScript
 imports when '--moduleResolution' is 'node16' or 'nodenext'.
```

There is a conversation on GitHub regarding this issue: https://github.com/microsoft/TypeScript/issues/49083.
The main solution suggested is to simply add the .js extension.